### DISCUSSION TITLE
Best Practices - Data Sharing Requirements for Charging Stations?

### DISCUSSION THREAD

**Discussion by jmandellaMTC** [openmobilityfoundation/curb-data-specification#121] [General]
This is a bit tangential and high-level, but my organization is seeking data sharing requirements to include in contracts with electric charging stations providers. In particular, best practice requirements regarding public APIs, automated charging station data collection, and overall interoperability with other devices/ITS, as reasonable. Any thoughts from the CDS community?

**Comment by schnuerle** [comment]
We've talked about this a lot at a few public working group meetings (at least 4). It's something on our radar and will likely be part of our Curb Objects discussions (3 meetings). Review the recordings, slides, and notes and let us know what you think!

https://github.com/openmobilityfoundation/curb-data-specification/wiki#meeting-agendas

**Comment by jmandellaMTC** [comment]
Hi Michael - thanks for following up! I attended these meetings and just now reviewed the materials you suggested for each to jog my memory.

CDS is certainly attempting to address my posted question head-on. Perhaps you can deduce this from your EV Charging Survey results, but it seems that the charging station industry has yet to coalesce behind a standard set of metrics that every charging station should gather and report out, since this industry is still ever-changing. Add to this the various layers of complexity such as the difference in purpose and necessary functionality in public vs. non-public chargers, networked vs. unnetworked chargers, etc, and it becomes obvious that my question might get quite different answers depending on who you ask.

As the CDS and EV Survey work progresses, I'll be very interested to see if certain minimum metrics for datasharing on EV charging infrastructure emerges (like the metrics proposed by the Center for Sustainable Energy in the attached picture).
Joel Mandella
Transportation/Climate Planner
***@***.******@***.***>
w (415) 778 6652
